### ISSUE TITLE
PKGBUILD fails

### ISSUE THREAD

**Issue by bernharl** [M-Reimer/wine-lol#99]
When building the AUR package I get the error message:
```
==> Starting build()...
  -> Building Wine-32...
checking build system type... x86_64-pc-linux-gnu
checking host system type... x86_64-pc-linux-gnu
checking whether make sets $(MAKE)... yes
checking for gcc... gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... configure: error: in `/home/bernhard/.cache/yay/wine-lol/src/wine-lol-32-build':
configure: error: cannot run C compiled programs.
If you meant to cross compile, use `--host'.
See `config.log' for more details
==> ERROR: A failure occurred in build().
    Aborting...
```

Running Arch Linux, updated today. I am on the mainline kernel 5.16.9

**Comment by Hack64** [comment]
Having the same issue, Arch Linux with kernel 5.16.9 (mainline or zen, doesn't make any difference). I suspect this has to do with the recent glibc update. I noticed that I couldn't start league anymore because wine-lol was asking for glibc 2.34:

```gamemodeauto: dlopen failed - /opt/wine-lol/lib32/libc.so.6: version `GLIBC_2.34' not found (required by /usr/lib32/libgcc_s.so.1)```

When trying to rebuild wine-lol against the newest glibc, i get the same error as above:

```
==> Starting build()...
  -> Building Wine-32...
checking build system type... x86_64-pc-linux-gnu
checking host system type... x86_64-pc-linux-gnu
checking whether make sets $(MAKE)... yes
checking for gcc... gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... configure: error: in `/home/hack64/.cache/yay/wine-lol/src/wine-lol-32-build':
configure: error: cannot run C compiled programs.
If you meant to cross compile, use `--host'.
See `config.log' for more details
==> ERROR: A failure occurred in build().
    Aborting...
 -> error making: wine-lol
```

**Comment by caseyjp11** [comment]
Same issues here.
Trying to play crashes with a libgcc_s.so.1 issue being reported by the wine client. This was after a normal system update. (I believe the GCC libs were updated today.)
As I had no issues building or playing prior, I have to assume this is a system update conflicting with league's installer/wine-lol.

I tried to reinstall, but the leagueoflegends-git script installation fails trying to build wine-lol.
